### DISCUSSION TITLE
Add a "Cancel" option.

### DISCUSSION THREAD

**Discussion by zzDave** [cline/cline#399] [Feature Requests]
There are times when its moving through a tasks workflow and you may want to cancel any further steps.

Example:
It makes changes to the code.
It now wants to inspect the website to verify changes.
At this point, something is wrong with my web server or I have it running on another port.  Instead of the agent trying to run this inspect command, I want to cancel and be finished with the task.

:)

**Comment by Ohmnitude** [comment]
Agree, verification often burns through money unnecessarily when we know the task is done. we should have [Accept] and [Cancel] because [Reject] is the same thing as saying try again which we often have to do by specifying how to proceed.

**Comment by saoudrizwan** [comment]
Added in v2!
